resources/content/schema/internal-test/internal-test-auth.json.d/register.json

```diff
@@ -0,0 +1,8 @@
+{
+    "authorize" : {
+        "register.key" : "cr",
+
+        "registrationToken.publicValue" : "",
+        "registrationToken.secretValue" : ""
```

Comment:
i don't think so. This has just been used from the earlier version as it used to exist.